python/spatialdata-integrity/src/spatialdata_integrity/cli.py

```
@@ -0,0 +1,145 @@
+                "element_type": e.element_type,
+                "element_name": e.element_name,
+                "is_valid": e.is_valid,
+                "chunks_checked": e.chunks_checked,
+                "errors": [
+                    {
+                        "chunk_index": list(e.chunk_index),
+                        "error_type": e.error_type,
+                        "error_message": e.error_message,
+                        "array_path": e.array_path,
+                    }
+                    for e in element.errors
```

Ruff (0.14.8)

Undefined name `element`

(F821)